src/lib/redux/cache.js

```diff
@@ -7,7 +7,7 @@ const HOUR_IN_MS = 3600000;
 const SERIALIZE_THROTTLE = 500;
 const MAX_AGE = 30 * DAY_IN_HOURS * HOUR_IN_MS;
 const STORAGE_KEY = 'REDUX_STATE';
-const SERIALIZED_STATE_VERSION = 2;
+const SERIALIZED_STATE_VERSION = 3;
```

Comment:
Previously, anything that changed the parsed endpoints list required either `localStorage.clear()` or a version bump.  It looks like this will be fixed in #40 :)